lib/improver/blending/spatial_weights.py

```diff
@@ -255,6 +255,80 @@ def normalised_masked_weights(weights_cube, blend_coord):
             result.append(weight_slice.copy(data=normalised_data))
         return result.merge_cube()
 
+    @staticmethod
+    def create_template_slice(cube_to_collapse, blend_coord):
```

Comment:
This function was created to change the dimensions in the output cube for the spatially varying weights. Before it returned a weights cube with the same dimensions as the cube_to_collapse. Now it returns a cube with dimensions of blend_coord, y x.
This was done to make the weighted blending code work better, and is a better representation of how the weights vary. (Before it may return a weights cube with dimensions of blend_coord, threshold, y, x even though the weights didn't vary along the threshold dimension.)
I haven't yet written unit tests for this, but ideas on what unit tests are needed are on the ticket.